transform/models/marts/fact_commits_monthly.sql

```diff
@@ -2,8 +2,8 @@ SELECT
   d.date_month, 
   c.repo_id, 
   COUNT(*) AS count_commits, 
-  lag(count_commits, 12) OVER (PARTITION BY c.repo_id ORDER BY d.date_month) AS last_year_count_commits, 
-  count_commits / last_year_count_commits AS yoy_growth, 
+  nullif((lag(count_commits, 12) OVER (PARTITION BY c.repo_id ORDER BY d.date_month)), 0) AS last_year_count_commits, 
```

Comment:
Nit: I think that we can do without the double `((` after nullif:

```
nullif(lag(count_commits, 12) OVER (PARTITION BY c.repo_id ORDER BY d.date_month), 0) AS last_year_count_commits,
```

And make nullif uppercase to keep coherent casing.